lighthouse-core/lib/network-request.js

```diff
@@ -297,12 +297,24 @@ module.exports = class NetworkRequest {
 
   /**
    * LR loses transfer size information, but passes it in the 'X-TotalFetchedSize' header.
+   * 'X-TotalFetchedSize' is the canonical transfer size in LR. Nothing should supersede it.
+   *
+   * The total length of the encoded data is spread out among multiple events. The sum of the
+   * values in onResponseReceived and all the onDataReceived events typically equals the value
+   * seen on the onLoadingFinished event. In <1% of cases we see the values differ. As we process
```

Comment:
fwiw in ~70% of cases in my local devtoolslog of a run I just did the final `encodedDataLength` differs from the sum of the events, not sure if I'm broken or the something else has changed :)

Reply:
interesting. could you or paul share your test scripts / LH code diff / devtool logs? We should control for what logs we are using + NS  + Chrome version so we know what's actually happening.

Reply:
oh I just looked at the last latest-run/defaultPass.devtoolslog.json I had, so it's probably fairly old